### PR TITLE
feat(Next.js): Added Spotlight option in server config

### PR DESCRIPTION
**Pull request by Shubhdeep12**
Added comment to add spotlight in Sentry.init for next.js server config

_Note: We did not include the installation and setup of Spotlight because it is not added by default, and it is also not mandatory._

#535

### CODE REVIEW

File: src/nextjs/templates.ts [hunk 1]
```diff
@@ -151,7 +160,9 @@ Sentry.init({
   tracesSampleRate: 1,
 
   // Setting this option to true will print useful information to the console while you're setting up Sentry.
-  debug: false,${additionalOptions}
+  debug: false,
+  ${additionalOptions}
+  ${spotlightOption}
```
Comment (lforst) [inline src/nextjs/templates.ts:165]: Looking at this change superficially I think this will mess up the output.

Comment (Shubhdeep12) [reply src/nextjs/templates.ts]: Hi @lforst this was adding some extra empty lines, which I just fixed.

Is there any specific issue that you are mentioning here? which I am missing?

**Review by lforst — APPROVED** [review]

File: src/nextjs/templates.ts (hunk 1 above)
Comment (lforst) [reply src/nextjs/templates.ts]: That was what I was referring to. Thanks for fixing it!

**Comment by Shubhdeep12** [comment]
> Hi, prettier is failing and also could you add a changelog entry? Thanks!

Hi @lforst it should be fixed now.

**Pull request merged**